### PR TITLE
 #172430232 setup app to deploy on heroku

### PR DESCRIPTION
**Pull request by izzett222**
#### What does this PR do?
setup our react app so that we can access it through heroku
#### Description of Task to be completed?
- add express server
- modify the webpack.config.prod
- add the build folder in gitignore
- add the start script
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[ #172430232](Deploy web application to Heroku)
#### Questions:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +25,12 @@
   },
   "homepage": "https://github.com/andela/devrepublic-bn-frontend#readme",
```
Comment (Ntare22) [inline package.json]: this link should change

**Review by Ntare22 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (izzett222) [reply package.json]: let me change it to stackup repo

**Review by Ntare22 — APPROVED** [review]

**Review by Baraka-Mugisha — APPROVED** [review]

**Review by bbaime98 — APPROVED** [review]

File: server.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const path = require('path');
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, 'build')));
+app.get('*', (req, res) => {
+  res.sendFile(path.join(__dirname, 'build', 'index.html'));
+});
+
+const PORT = process.env.PORT || 4000;
```
Comment (jabichris) [inline server.js:13]: Do you have any idea what will happen if you remove 4000? Please make sure your dotenv is well set.

File: server.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const express = require('express');
```
Comment (jabichris) [inline server.js]: Make sure your `babel` is well set so that you can use ES6 Import.

Comment (izzett222) [reply server.js]: i setup babel, now import can be used

File: server.js (hunk 2 above)
Comment (izzett222) [reply server.js:13]: I added the dotenv package, the PORT can be read from the env file

**Pull request merged**